main.go

```diff
@@ -157,9 +171,17 @@ func completer(d prompt.Document) []prompt.Suggest {
 		}
 
 		if command == "pull" {
+			if strings.Index(word, ":") != -1 || strings.Index(word, "@") != -1 {
```

Comment:
I've found a bug and tried to solve that. If the user entered a tag or digest, the search was running.